internal/pipe/git/git.go

```diff
@@ -32,7 +32,9 @@ func (Pipe) Run(ctx *context.Context) error {
 		return err
 	}
 	ctx.Git = info
-	log.Infof("releasing %s, commit %s", info.CurrentTag, info.Commit)
+	if !ctx.Snapshot {
+		log.Infof("releasing %s, commit %s", info.CurrentTag, info.Commit)
```

Comment:
I agree that _releasing_ is not relevant here, but wouldn't it be still useful to output the commit SHA as a confirmation of what exact SHA did GoReleaser pick up?